src/game/editor/popups.cpp

```diff
@@ -2590,12 +2590,43 @@ CUI::EPopupMenuFunctionResult CEditor::PopupSwitch(void *pContext, CUIRect View,
 {
 	CEditor *pEditor = static_cast<CEditor *>(pContext);
 
-	CUIRect NumberPicker, FindEmptySlot;
+	CUIRect NumberPicker, FindEmptySlot, EmptyRow, ViewEmptySlot;
 
 	View.VSplitRight(15.0f, &NumberPicker, &FindEmptySlot);
 	NumberPicker.VSplitRight(2.0f, &NumberPicker, nullptr);
-	FindEmptySlot.HSplitTop(13.0f, &FindEmptySlot, nullptr);
+
+	FindEmptySlot.HSplitBottom(13.0f * 2, &FindEmptySlot, &EmptyRow);
+	EmptyRow.HSplitBottom(13.0f, &EmptyRow, &ViewEmptySlot);
+
```

Comment:
```suggestion
	FindEmptySlot.HSplitTop(13.0f, &FindEmptySlot, &ViewEmptySlot);
	ViewEmptySlot.HSplitTop(13.0f, nullptr, &ViewEmptySlot);
```

And remove the variable `EmptyRow`